lib/js/src/protocol/SdlProtocolBase.js

```diff
@@ -495,27 +493,34 @@ class SdlProtocolBase {
         this._sdlProtocolListener.onRpcMessageReceived(rpcResponse);
     }
 
-
     /**
-     * Ends a specific service
-     * @param {ServiceType} serviceType - Service type being ended. When the RPC service is ended the entire session ends.
-     * @param {Number} sessionId - represents a byte 
+     * Ends a session.
      */
-    endService (serviceType, sessionId) {
+    endSession () {
```

Comment:
Java Suite defines this method with arguments for a `sessionId` and a `hashId`. Please confirm with @joeygrover that it is OK to use the instance's private properties rather than supporting passed arguments.

Reply:
Updating description. Right now it ends the default session and that should be good for what we need right now. There are no other sessions to end. endService can still be used to an a specific audio or video service.